qiskit_experiments/library/characterization/fine_amplitude.py

```diff
@@ -99,43 +98,75 @@ def _default_experiment_options(cls) -> Options:
                 by doing :code:`options.gate_class()`.
             normalization (bool): If set to True the DataProcessor will normalized the
                 measured signal to the interval [0, 1]. Defaults to True.
-            add_sx (bool): If True then the circuits will start with an sx gate. This is typically
-                needed when calibrating pulses with a target rotation angle of :math:`\pi`. The
-                default value is False.
-            add_xp_circuit (bool): If set to True then a circuit with only an X gate will also be
-                run. This allows the analysis class to determine the correct sign for the amplitude.
+            add_cal_circuits (bool): If set to True then two circuits to calibrate 0 and 1 points
+                will be added. These circuits are often needed to properly calibrate the amplitude
+                of the ping-pong oscillation that encodes the errors.
         """
         options = super()._default_experiment_options()
-        options.repetitions = list(range(15))
+        options.repetitions = list(range(1, 15))
         options.gate = None
         options.normalization = True
-        options.add_sx = False
-        options.add_xp_circuit = True
+        options.add_cal_circuits = True
```

Comment:
What happens if you disable this? Curve analysis may crash since it tries to fit for null array. I think I'm going to overhaul the analysis to make it more flexible. For now I think you can remove this option if disabling this cause the problem. I think there is no disadvantage for having this option.